normandy/recipes/tests/__init__.py

```diff
@@ -44,7 +44,7 @@ class Meta:
     @factory.post_generation
     def signed(self, create, extracted=False, **kwargs):
```

Comment:
A comment somewhere in here describing why you're using the canonical JSON for the signature would be useful.